Testnet/NonFungibleToken/NonFungibleToken/NonFungibleToken.cs

```diff
@@ -364,12 +330,12 @@ private void TransferInternal(Address to, ulong tokenId)
     /// <remarks>Use and override this function with caution. Wrong usage can have serious consequences.</remarks>
     /// <param name="from">Address from wich we want to remove the NFT.</param>
     /// <param name="tokenId">Which NFT we want to remove.</param>
-    private void RemoveNFToken(Address from, ulong tokenId)
+    private void RemoveToken(Address from, ulong tokenId)
     {
         Assert(GetIdToOwner(tokenId) == from);
         var tokenCount = GetBalance(from);
         SetBalance(from, checked(tokenCount - 1));
-        State.Clear(GetIdToOwnerKey(tokenId));
+        SetIdToOwner(tokenId, Address.Zero); //Work around for clear bug State.Clear(GetIdToOwnerKey(tokenId));
```

Comment:
Nitpick - Remove this comment referencing a bug. If this is a mainnet ready contract, don't give any indication of issues or workarounds. Rather if a comment is needed just use `// clear tokenId state`